packages/backend/src/controllers/gitIntegrationController.ts

```diff
@@ -29,23 +29,6 @@ import { BaseController } from './baseController';
 @Response<ApiErrorPayload>('default', 'Error')
 @Tags('Git Integration')
 export class GitIntegrationController extends BaseController {
-    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
-    @SuccessResponse('200', 'Success')
-    @Get('/')
```

Comment:
This was used in sql runner, I replaced this endpoint with a projectless endpoint /config

![Screenshot from 2025-02-17 11-34-20](https://github.com/user-attachments/assets/20744dfb-1033-4745-9e67-e3f9dbf9e4c5)

![image](https://github.com/user-attachments/assets/9e55c18b-ae1e-42da-99c4-7c734a364f1d)